parts/k8s/cloud-init/nodecustomdata.yml

```diff
@@ -126,6 +126,15 @@ write_files:
   content: !!binary |
     {{CloudInitData "etcIssue"}}
 
+{{if not .IsVHDDistro}}
```

Comment:
@CecileRobertMichon thoughts on this pattern?

Reply:
just one thought, do we want this for Coreos? If not `.IsUbuntu` might be a better fit

Reply:
The idea is to not use cloud-init for VHD-backed flows, because we've already done this work via packer.

Reply:
yes but my question is should we also add this file to coreos VMs?

Reply:
CoreOS is out of scope for CIS at this time

Reply:
the way you're paving the file right now it IS being added to CoreOS VMs. What I'm saying is that maybe we should exclude CoreOS as well to not change that scenario (and use `.IsUbuntu` instead).

Reply:
I see what you're saying.